### ISSUE TITLE
Add additional status

### ISSUE THREAD

**Issue by Hellowlol** [fondberg/easee#25]
If auth is required to use the charger, in in the app, its called waiting for authorization, however in the api the data that is sent has 'chargerOpMode': 2 (PAUSED)

I think we should check for this add add a waiting for approval or authorization status.

Also we dont use the levelOfAccess that /api/chargers/{id} returns. There is no way to know what kind of access is required. We can check if auth is required using charger.get_config()["authorizationRequired"] but thats it.

If this is something that you want, i can a PR for this.

**Comment by tmjo** [comment]
Did you check if this info is available in the sensor "reason_for_no_current"?

**Comment by Hellowlol** [comment]
There is no sensor in this library, but it isnt available in hass_easee

**Comment by tmjo** [comment]
Right, of course, I forgot this was for the API. But the is still available, did you check what the reason for no current value returns?

**Comment by Hellowlol** [comment]
I havnt checked that out. Ill do that

**Comment by Hellowlol** [comment]
When the charger require auth, the reason_for_no_current is None. So no help there :(
